jobs/views.py

```diff
@@ -38,7 +38,11 @@ class JobCreate(CreateView):
     success_url = reverse_lazy('jobs:jobs')
 
     def form_valid(self, form):
-        company, created = Company.objects.get_or_create(name=form.cleaned_data['company_name'])
-        form.instance.company = company
-        form.instance.created = timezone.now()
+        job = form.save(commit=False)
+        company, created = Company.objects.get_or_create(
+            name=form.cleaned_data['company_name'],
+            website=form.cleaned_data['website']
+        )
```

Comment:
include the website as well when doing the get_or_create.  This should also avoid the case where there are two companies with the same name as we don't want the get to return more than 1.